### PR TITLE
Fix Issue 897 - logging for membersrvc aligned with peer logging

### LINKED ISSUE #897: Incorporate logging with different granularities
Implement different log granularity for different parts of the CA (ECA, TCA, TLSCA)

### PR DESCRIPTION
**Pull request by christo4ferris**
## Description

aligned logging in membersrvc with that in peer. 
Moved logging from core to its own package so it could be shared, renamed flogging.
## Motivation and Context

Fixes #897 
## How Has This Been Tested?

ran unit tests and behave... no tests added 
## Checklist:

- [x] I have added a [Signed-off-by](https://github.com/hyperledger/fabric/blob/master/CONTRIBUTING.md#legal-stuff).
- [x] I have either added documentation to cover my changes or this change requires no new documentation.
- [x] I have either added unit tests to cover my changes or this change requires no new tests.
- [x] I have run [golint](https://github.com/golang/lint) and have fixed valid warnings in code I have added or modified. This tool generates false positives so you may choose to ignore some warnings. The goal is clean, consistent, and readable code.

Signed-off-by: Christopher Ferris chrisfer@us.ibm.com

### CODE REVIEW

**Comment by JonathanLevi** [comment]
@christo4ferris: Thank you vm!
Looks good/great to me (including the scope of the constant string. Did not go unnoticed ;-))

**Comment by srderson** [comment]
@christo4ferris Looks like there's a merge conflict on this PR now. Sounds like it's ready to merge once you rebase on master.

**Comment by christo4ferris** [comment]
@srderson conflicts resolved assuming travis and jenkins gods are appeased, ready to merge

File: membersrvc/ca/ca.go [hunk 1]
```diff
@@ -576,8 +581,8 @@ func (ca *CA) validateAndGenerateEnrollID(id, affiliation string, role pb.Role)
 func (ca *CA) registerUser(id, affiliation string, role pb.Role, registrar, memberMetadata string, opt ...string) (string, error) {
 	memberMetadata = removeQuotes(memberMetadata)
 	roleStr, _ := MemberRoleToString(role)
-	Trace.Printf("Received request to register user with id: %s, affiliation: %s, role: %s, registrar: %s, memberMetadata: %s\n",
-		id, affiliation, roleStr, registrar, memberMetadata)
+	caLogger.Debugf("Received request to register user with id: %s, affiliation: %s, role: %s, registrar: %s, memberMetadata: %s\n",
+		id, affiliation, affiliationRole, roleStr, registrar, memberMetadata)
```
Comment (juliancarrivick-ibm) [inline membersrvc/ca/ca.go]: FYI the tests failed here because affiliationRole isn't defined (I assume you meant to rename roleStr)

**Comment by christo4ferris** [comment]
migrated PR to Gerrit, closing

**Pull request closed without merging**